block-builder/src/main.rs

```diff
@@ -1,30 +1,15 @@
 use openrank_block_builder as block_builder;
-use openrank_sequencer as sequencer;
 use std::error::Error;
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn Error>> {
     let mut bb = block_builder::Node::init().await?;
-    let mut sequencer = sequencer::Node::init().await?;
 
-    // Recover the state from the local DB
-    bb.node_recovery()?;
```

Comment:
I think, even though we remove `JobCoordinator`, we still need this `node_recovery` logic.
Why do we remove this `node_recovery` logic from `block-builder`?

Reply:
What should this logic do?

Reply:
Original `block_builder::node_recovery` stores the `JobResult` in the DB.
We don't need that logic any more?

Reply:
As discussed in the meeting. It was a depricated logic